common/scala/src/main/scala/whisk/core/entity/Exec.scala

```diff
@@ -96,7 +96,7 @@ protected[core] case class BlackBoxExec(image: String) extends Exec(Exec.BLACKBO
 /**
  * add temporary field that holds the "fixed" names for components where the '_' is replaced by the user's namespace
  */
-protected[core] case class SequenceExec(code: String, components: Vector[String], fixedComponents: Option[Vector[String]] = None) extends Exec(Exec.SEQUENCE) {
+protected[core] case class SequenceExec(code: String, components: Vector[String], fixedComponents: Option[Vector[FullyQualifiedEntityName]] = None) extends Exec(Exec.SEQUENCE) {
```

Comment:
Can you explain to me again why we need fixedComponents and components? When you fully resolve the names, I don't see why the existing implementation can't accept those instead. I would view this as a good checkpoint/commit point that should be semantic preserving.